### DISCUSSION TITLE
Ubuntu 22.04 issues

### DISCUSSION THREAD

**Discussion by xueweiwujxw** [flameshot-org/flameshot#3434] [Q&A] [answered]
Why is it that when I use screenshots under Ubuntu 22.04, the default is to capture the full screen, and only share it? If I cancel, it will prompt a capture error.
![截图 2023-12-07 16-14-21](https://github.com/flameshot-org/flameshot/assets/45992640/47b87a73-3db9-4000-836d-9d0adb4e0aba)

**Comment by xueweiwujxw** [comment]
It seems to be a problem with wayland. Can it be optimized for wayland?

**Comment by xueweiwujxw** [comment, marked as answer]
I found your [Help Page](https://flameshot.org/docs/guide/wayland-help/#gnome).
Maybe this is a [solution](https://www.cnblogs.com/pipci/p/16995478.html) or a compromise.

**Reply by xueweiwujxw** [reply]
The decision made by the gnome developers is a piece of shit and I'm sorry for wasting your time

**Comment by mmahmoudian** [comment]
This is Gnome Wayland issue and not Ubuntu. If you are annoyed by it, either give feedback to Gnome people or change Gnome and use something like KDE Plasma.

**Reply by xueweiwujxw** [reply]
I disabled wayland and used X11, and it worked.

**Reply by mmahmoudian** [reply]
That is not a correct way to go. Most Linux distros are dropping support for X11. If this issue is that annoying for you, I suggest changing Gnome to something better rather than disabling Wayland.

**Reply by xueweiwujxw** [reply]
Oh. Thanks a lot. I will try.
